.github/actions/process-test-results/action.yaml

```diff
@@ -56,6 +56,7 @@ runs:
       uses: EnricoMi/publish-unit-test-result-action@v2
       if: always()
       with:
+        check_name: ${{ inputs.test_group_name }}
```

Comment:
I'll be interested to see what the comment looks like. Are the values of `test_group_name` reasonable? would they make sense if it shows up as a comment on a pr?